cartographer/io/xray_points_processor.cc

```diff
@@ -81,10 +81,9 @@ Image IntoImage(const PixelDataMatrix& mat) {
       double mix_g = Mix(1., mean_g_in_column, saturation);
       double mix_b = Mix(1., mean_b_in_column, saturation);
 
-      const uint8_t r = common::RoundToInt(mix_r * 255.);
-      const uint8_t g = common::RoundToInt(mix_g * 255.);
-      const uint8_t b = common::RoundToInt(mix_b * 255.);
-      image.SetPixel(x, y, {{r, g, b}});
+      image.SetPixel(x, y,
+                     {{static_cast<float>(mix_r), static_cast<float>(mix_g),
```

Comment:
Any reason to do the computations for `mix_?` in `double` and cast here? Seems the computations could be done in `float` without any problem.